src/Microsoft.Graph/Generated/requests/ISiteReferenceRequest.cs

```diff
@@ -18,19 +18,6 @@ namespace Microsoft.Graph
     /// </summary>
     public partial interface ISiteReferenceRequest : IBaseRequest
     {
-        /// <summary>
-        /// Deletes the specified Site reference.
-        /// </summary>
-        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
-        /// <returns>The task to await.</returns>
-        System.Threading.Tasks.Task DeleteAsync(CancellationToken cancellationToken = default);
```

Comment:
This looks to be a breaking change. Most like caused by https://github.com/microsoftgraph/msgraph-metadata/pull/141/files